woocommerce/woocommerce-ios
# Enabled Products M5 features for all the users

Closes #3400 

## Description
In this PR and in accordance with our [discussion](https://a8c.slack.com/archives/CGPNUU63E/p1608386588040300), we decided to release to all our users the new functionalities under the Products M5. I've removed all the m5 feature flag from our codebase, and update the unit tests in accordance.
**The new functionalities include:**
- Add and edit Linked Products.
- Add and edit Downloadable Files.
- Delete a product.

**Important**: this PR should be merged when we complete the implementation of the tracking events. I'll take care of the missing events.

## Testing
We should test that all the functionalities described above are visible to all our users. As described in the slack discussion, we enabled the functionalities without adding them under the beta screen in the app settings (like we did in the past).
Therefore:
- Make sure that you are able to see the Linked Products row under a product detail.
- Make sure that you are able to see the Downloadable Files flag in Product Settings, and that if you turn on the flag you are able to see the Downloadable Files row under the product detail.
- Make sure that you are able to delete a product.


Update release notes:
